fakenet/configs/default.ini

```diff
@@ -215,16 +236,18 @@ UseSSL:      No
 Timeout:     10
 ProcessWhiteList: ncat.exe, nc.exe
 HostBlackList: 5.5.5.5
+Hidden:      False
 
 [DNS Server]
 Enabled:     True
 Port:        53
 Protocol:    UDP
 Listener:    DNSListener
-ResponseA:   192.0.2.123
+ResponseA:   192.168.105.131
```

Comment:
Should this still be 192.0.2.123?

Reply:
Yeah, it's the example class C known as TEST-NET and it serves to thwart malware that detects DNS replies of 127.x.x.x. Users are of course free to override it, though.